packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2029,16 +2064,25 @@ describe('plugin-meetings', () => {
                   muted: false,
                   underlyingTrack: webrtcAudioTrack2
                 };
+                const fakeMicrophoneStream = {
+                  on: sinon.stub(),
+                  off: sinon.stub(),
+                  outputStream:{
+                    getTracks: () => {
+                      return [fakeMicrophoneTrack2]
+                    }
+                  }
+                }
 
-                await meeting.publishTracks({microphone: fakeMicrophoneTrack2});
+                await meeting.publishStreams({microphone: fakeMicrophoneStream});
                 await stableState();
 
                 // only the roap media connection should be updated
                 if (isMultistream) {
-                  assert.calledOnceWithExactly(fakeMultistreamRoapMediaConnection.publishTrack, fakeMicrophoneTrack2);
+                  assert.calledOnceWithExactly(meeting.sendSlotManager.publishStream, MediaType.AudioMain, fakeMicrophoneStream);
```

Comment:
in these semi-integration tests we should be checking that correct methods on fakeMultistreamRoapMediaConnection or fake send slots are called, not on the SDK's sendSlotManager

Reply:
Have commented about it [here](https://github.com/webex/webex-js-sdk/pull/3053#discussion_r1319522326).